site/content/docs/5.2/forms/overview.md

```diff
@@ -164,6 +164,30 @@ Use the `.is-required` class on `label` to add a `*` symbol after it.
 </form>
 {{< /example >}}
 
+## Form helper
+
+Form helper is a button using `.form-helper` that provides extra help information to a form control. Be sure to include text for screen readers with a visually hidden text.
+
+Depending on the extra help information content, choose wisely between triggering a [popover]({{< docsref "/components/popovers" >}}) or a [tooltip]({{< docsref "/components/tooltips" >}}). Tooltips should be used for small and simple texts without any actionable elements inside, while popovers are used in other cases.
+
+This form helper should be displayed right after the label.
+
+{{< example >}}
+<label for="tooltipTextInput" class="form-label">Input with helper inside a Popover</label>
+<button type="button" class="form-helper" data-bs-toggle="popover" data-bs-placement="top" data-bs-content="Help for input">
+  <span class="visually-hidden">Helper for input</span>
+</button>
+<input type="text" id="tooltipTextInput" class="form-control" placeholder="Tooltip input">
+
+<label for="tooltipSelect" class="form-label is-required mt-3">Select menu with helper inside a Tooltip</label>
```

Comment:
Seeing this `.mt-3` here, I'm wondering if we shouldn't do the following:
```html
<div class="mb-3">
   <!-- First example -->
</div>
<div>
   <!-- Second example -->
</div>
```

to have consistency with the other forms in the docs. But might be dumb. You'll tell me.